langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -380,6 +556,11 @@ public isolated function fromString(string s) returns xml|error = @java:Method {
 
 # Selects all the items in a sequence that are of type `xml:Text`.
 #
+# ```ballerina
+# xml books = xml `<book></book>Hamlet<book></book>Play`;
```

Comment:
Why have we used `<book></book>` here instead of `<book/>` or an element with content?